### PR TITLE
docs: workflow status on dashboard

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### CODE REVIEW

File: docs/check-workflow-status.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
 title: Check Workflow Status
 ---
 
-<!-- ## Check workflow status using Chaos Dashboard -->
+## Check workflow status using Chaos Dashboard
```
Comment (en-jin19) [inline docs/check-workflow-status.md:5]:
```suggestion
## Check Workflow status using Chaos Dashboard
```

File: docs/check-workflow-status.md [hunk 2]
```diff
@@ -2,9 +2,15 @@
...
 
-<!-- TODO: This section will be added after the optimization of Chaos Dashboard completes. -->
+1. List all the workflows in the Chaos Dashboard.
+
+![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)
+
+2. Select the workflow you want to check, show the details of the workflow.
+
+![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)
```
Comment (en-jin19) [inline docs/check-workflow-status.md:13]:
```suggestion
1. List all Workflow in Chaos Dashboard.

![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)

2. Select the Workflow you want to check the status, see the details of the Workflow.

![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)
```

File: docs/check-workflow-status.md [hunk 3]
```diff
@@ -2,9 +2,15 @@
...
 
 ## Check workflow status using `kubectl` commands
```
Comment (en-jin19) [inline docs/check-workflow-status.md:15]:
```suggestion
## Check Workflow status using `kubectl` commands
```

File: i18n/zh/docusaurus-plugin-content-docs/current/check-workflow-status.md [hunk 4]
```diff
@@ -2,9 +2,15 @@
...
 
-<!-- ## 通过 Chaos Dashboard 检查 Workflow 状态 -->
+## 通过 Chaos Dashboard 检查 Workflow 状态
 
-<!-- TODO: 待 Chaos Dashboard 完善后补充这一部分 -->
+1. 在 Chaos Dashboard 中列出所有的 Workflow。
+
+![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)
+
+2. 选择你想要检查的 Workflow，显示 Workflow 的详细信息。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/check-workflow-status.md:11]:
```suggestion
2. 选择你想要检查的 Workflow，点击 Workflow 的详细信息。
```

File: versioned_docs/version-2.1.1/check-workflow-status.md [hunk 5]
```diff
@@ -2,9 +2,15 @@
...
+## Check workflow status using Chaos Dashboard
 
-<!-- TODO: This section will be added after the optimization of Chaos Dashboard completes. -->
+1. List all the workflows in the Chaos Dashboard.
+
+![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)
+
+2. Select the workflow you want to check, show the details of the workflow.
+
+![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)
```
Comment (en-jin19) [inline versioned_docs/version-2.1.1/check-workflow-status.md:13]:
```suggestion
1. List all workflows in Chaos Dashboard.

![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)

2. Select the workflow you want to check and see the details of the workflow.

![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)
```

**Comment by en-jin19** [comment]
@STRRL could you please invite a tech reviewer? thanks!

**Comment by STRRL** [comment]
> @STRRL could you please invite a tech reviewer? thanks!

Updated PTAL

**Comment by STRRL** [comment]
PTAL

/cc @g1eny0ung @cwen0

**Review by en-jin19 — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@STRRL CI was broken! Can you fix it?

**Comment by STRRL** [comment]
> @STRRL CI was broken! Can you fix it?

Updated! PTAL!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**